release.py

```diff
@@ -65,11 +66,11 @@ def is_catkin_package():
 
 def github_repo_exists(url):
     try:
-        check_call(['git', 'ls-remote', url], IGNORE_DRY_RUN)
+        check_call(['git', 'ls-remote', '-q', '--exit-code', url], IGNORE_DRY_RUN)
```

Comment:
I tried a dry-run here and got this error:

```
Dry-run running:
  git ls-remote --exit-code --heads https://github.com/ignition-release/ign-physics2-release main

Error running command (git ls-remote --exit-code --heads https://github.com/ignition-release/ign-physics2-release main).
stdout: b''
stderr: b''
```

Reply:
I think this error message is printed because `ign-physics2-release` doesn't have a `main` branch yet. The following should improve the printing of stdout and stderr:

~~~
diff --git a/release.py b/release.py
index 4c5bbc3a..b1b02c76 100755
--- a/release.py
+++ b/release.py
@@ -372,8 +372,8 @@ def check_call(cmd, ignore_dry_run = False):
                 raise ErrorNoUsernameSupplied()
             # Unkown exception
             print('Error running command (%s).'%(' '.join(cmd)))
-            print('stdout: %s'%(out))
-            print('stderr: %s'%(err))
+            print('stdout: %s'%(out.decode()))
+            print('stderr: %s'%(err.decode()))
             raise Exception('subprocess call failed')
         return out, err
 
~~~

Reply:
db8ae05105679c41d4dee61de7c46f61146f9423